Update agent setup instructions (#94)
neo4j/docs-ops-manager: modules/ROOT/pages/installation/docker/container.adoc

= Server Docker container
:description: This page describes how to run a NOM server in a Docker container.

For every NOM release, a Docker image for the NOM server is published to the https://hub.docker.com/r/neo4j/neo4j-ops-manager-server/tags[Docker Hub verified repository].

> == Getting the NOM server Docker image
> Latest NOM server image for the minor release can be pulled from the verified repository for NOM server

Reviewer: ```suggestion
== Getting the NOM server Docker image
The latest NOM server image can be pulled from the verified repository by running the following command.
```
